LICENSE.txt

```diff
@@ -602,12 +602,6 @@ THE SOFTWARE.
 
 --------------------------------------------------------------------------------
 
-3rdparty dependency github.com/apache/arrow-go/v18
-is statically linked in certain binary distributions, like the Python wheels.
-github.com/apache/arrow-go/v18 is under the Apache-2.0 license.
-
---------------------------------------------------------------------------------
-
 3rdparty dependency github.com/apache/thrift/lib/go/thrift
 is statically linked in certain binary distributions, like the Python wheels.
 github.com/apache/thrift/lib/go/thrift is under the Apache-2.0 license.
```

Comment:
We can remove this because apache/thrift is an ASF project: https://infra.apache.org/licensing-howto.html#bundle-asf-product